### PR TITLE
signin and signup with api

### PR DESCRIPTION
**Pull request by Asifrizwan6989**
Signin and Signup page created, Api connection working fine

Server-side url: https://github.com/Asifrizwan6989/AutoZoneServer.git

### CODE REVIEW

File: autozone-poc/src/components/Signin/Signin.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   
+      <div className="container h-100">
+        <div className="row h-100 align-items-center">
+          <div className="col-4"></div>
+          <div className="col-4 align-items-center signincontainer">
+            <div className="page-title text-center">
+              <h2 className="signinhead">Sign In</h2>
+              
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <input className="form-control" type="email" name="emailid" id="emailid" placeholder="Enter email id" ref={register({required:true})}/>
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Signin/Signin.js]: Please discuss with Gopal and resuse the input field component

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]
Where is the unit test case

File: autozone-poc/src/components/Signin/Signin.js (hunk 1 above)
Comment (Asifrizwan6989) [reply autozone-poc/src/components/Signin/Signin.js]: implemented and push in second commit

**Comment by gopalprathi** [comment]
@Asifrizwan6989 Resolve conflicts

**Comment by gokulwebdeveloper** [comment]
Please resolve the conflicts in the branch

**Comment by Asifrizwan6989** [comment]
changes done in signin and signup pull request

**Pull request closed without merging**